src/Caching/Impl/TypeNames.cs

```diff
@@ -101,7 +100,12 @@ internal static class TypeNames {
                         parts.MemberNames = typeName == "..." ? new[] { "ellipsis" } : typeName.Split('.').ToArray();
                         break;
                     default:
-                        parts.ModuleName = typeName;
+                        if (typeName.EndsWith("(stub)")) {
```

Comment:
Do we have test for this?

Reply:
Yes, some of `LibraryModulesTests` would fail otherwise.